src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,46 @@
 package com.theironyard.novauc;
 
+import com.sun.tools.internal.xjc.reader.xmlschema.bindinfo.BIConversion;
+import org.eclipse.jetty.server.Authentication;
+import spark.ModelAndView;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.HashMap;
+
 public class Main {
 
+        static User user;
+
     public static void main(String[] args) {
-	// write your code here
+        // write your code here
+
+        Spark.init();
+
+
+        Spark.get(
+                "/",
+                ((request, response) -> {
+                    HashMap m = new HashMap();
+                    if (user == null) {
+                        return new ModelAndView(m, "index.html");
+                    } else {
+                        m.put("name", user.name);
+                        return new ModelAndView(m, "index.html");
+                    }
```

Comment:
I see your note.  We need to do a one on one Monday.  We can talk about the assignment then.